tests/utils/structure_tester.py

```diff
@@ -26,88 +28,155 @@
 )
 
 
+def get_enabled_prompt_drivers(prompt_drivers_options) -> list[BasePromptDriver]:
```

Comment:
Is there a better place to put this? Can't put it in the class as a `@staticmethod` because `StructureTester` does not exist yet. Putting it out here prevents me from using the `TesterPromptDriverOption` type hint.

I want to keep the fields static.

Reply:
would it make sense as a class method?

Reply:
Same issue: `"StructureTester" is not defined`